index.html

```diff
@@ -5,17 +5,200 @@
+                <br>
+                <button class="try-it">Try It</button>
+            </div>
+        </section>
+
+        <section class="juice-bar">
+            <h2 class="fade-in">Get Juice Bar</h2>
+            <div class="bar-container">
+                <div class="juice01 fade-in">
+                    <h2>1</h2>
+                    <h3>Local Delivery</h3>
+                    <p>I'm a paragraph. Click here to add your own text and edit me.</p>
+                </div>
+                <div class="juice02 fade-in">
+                    <h2>2</h2>
+                    <h3>In Store Pick Up</h3>
+                    <p>I'm a paragraph. Click here to add your own text and edit me.</p>
+                </div>
+                <div class="juice03 fade-in">
+                    <h2>3</h2>
+                    <h3>Curbside Pick Up</h3>
+                    <p>I'm a paragraph. Click here to add your own text and edit me.</p>
+                </div>
+            </div>  
+        </section>
 
-    <!-- Add your markup here -->
+   </main>
+   <footer>
+        <h2 class="fade-in">Follow Us</h2>
+        <a href="" class="fade-in">@juicebar</a>
+        <div class="footer-container">
+            <div class="footer-pics fade-in">
+                <div class="footer-img1"><img src="/Users/Mac/Documents/CYF/HTML-CSS-Coursework-Week4/img/IMG_0441.webp"/></div>
+                <div class="footer-img2"><img src="/Users/Mac/Documents/CYF/HTML-CSS-Coursework-Week4/img/CHOSEN_IMG_0331.webp"/></div>
+                <div class="footer-img3"><img src="/Users/Mac/Documents/CYF/HTML-CSS-Coursework-Week4/img/IMG_0715.webp"/></div>
+                <div class="footer-img4"><img src="/Users/Mac/Documents/CYF/HTML-CSS-Coursework-Week4/img/CHOSEN_IMG_0066.webp"/></div>
+            </div>
+            <div class="column-1">
+                <h3>Get Started</h3>
+                <ul>
+                    <li>Home</li>
+                    <li>Story</li>
+                    <li>Menue</li>
+                    <li>Order online</li>
+                    <li>Locations & Hours</li>
+                    <li>Contact</li>
+                </ul>
+            </div>
+            <div class="column-2">
+                <h3>Connect</h3>
+                <ul>
+                    <li>Instagram</li>
+                    <li>Facebook</li>
+                    <li>Youtube</li>
+                </ul>
+            </div>
+            <div class="column-3">
+                <h3>Join Our Mailing List</h3>
+                <label for="email" class="email-label">Email</lable>
+                <input type="email" id="email">
+                <button class="submit">Submit</button>
+            </div>
+            <div class="column-4">
+                <h3><a href="">Serving You Safely :)</a></h3>
```

Comment:
Can we use the HTML smiley code here https://www.w3schools.com/charsets/ref_emoji_smileys.asp